app/services/auth_helper.rb

```diff
@@ -33,73 +32,4 @@ def first_with_role(role)
       end
     end
   end
-
```

Comment:
This was authentication code for tests that didn't belong on the production side. It moved to a module in the test tree.